serving/docs/lmi/user_guides/lmi_input_output_schema.md

```diff
@@ -1,7 +1,7 @@
 # LMI handlers Inference API Schema
 
 This document provides the default API schema for the inference endpoints (`/invocations`, `/predictions/<model_name>`) when using the built-in inference handlers in LMI containers.
-This schema is applicable to our latest release, v0.27.0.
+This schema is supported from version 0.27.0.
```

Comment:
We added info like TGI compat and SSE in streaming in the doc, only applicable to 0.28.0. Should we say that somewhere?

Reply:
oh right, good callout. I might change this to say applicable to our latest release, please see <github link> for previous releases